packages/cspell-eslint-plugin/assets/options.schema.json

```diff
@@ -326,6 +326,19 @@
       },
       "type": "object"
     },
+    "cspellOptionsRoot": {
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "format": "uri",
+          "type": "string"
+        }
+      ],
+      "description": "Specify the root path of the cspell configuration. It is used to resolve `imports` found in  {@link  cspell } .\n\nexample: ```js cspellOptionsRoot: import.meta.url // or cspellOptionsRoot: __filename ```",
```

Comment:
You describe how to could configure it, but not what it could be used to.

I know the PR and I don't get it.

I think you should bring some examples about what it allows to do

Reply:
I was struggling with that.

It is used to tell the spell checker how to resolve imports.

**`eslint.config.js`**

```js
  rules: {
      '@cspell/spellchecker': [
          'error',
          {
                cspellOptionsRoot: import.meta.url,
                cspell: {
                    import: ['./cspell.config.yaml', '@cspell/dict-de-de'],
          },
      ],
  },
```

It needs to search for two imports: `./cspell.config.yaml` and `@cspell/dict-de-de`.

Assume `import.meta.url` is `file:///User/ci/project/code/eslint.config.js`.

- `./cspell.config.yaml` will be resolved to be `file:///User/ci/project/code/cspell.config.yaml`
- `@cspell/dict-de-de` will search from `/User/ci/project/code/` looking for `node_modules` containing `@cspell/dict-de-de`. It might resolve to be `file:///User/ci/project/node_modules/@cspell/dict-de-de/cspell-ext.json`

Reply:
It's definitely uneasy to explain especially in a conf file.

So maybe it's worth a page on the documentation and here you simply mention the link to the documentation